images/builders/mlflow/mlflow/mlflow_run.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+import argparse
+import mlflow
+import os
+from mlflow.projects.utils import load_project
+
+
+def run_mlflow_project(workdir: str, entrypoint: str, experiment: str, artifact_subpath: str, save_result_to_file: str) -> None:
+    """Run an MLproject entrypoint with arguments read from
+    FuseML workflow input parameters and save the result
+    artifact URL into the specified file.
+
+    Args:
+        workdir (str): path to the directory where the MLproject file resides
+        entrypoint (str): MLFlow project entrypoint to execute
+        experiment (str): MLFlow experiment name
+        artifact_subpath (str): path relative to the the MLFlow project run output
+                                artifact storage location to return as result
+        result_file (str): location of file where to save the URL of the output artifact
+    """
+    import os.path
```

Comment:
Is there a reason to import `os.path` here rather than on top of the file?

Reply:
no, just an overlook; fixed it